index.js

```diff
@@ -0,0 +1,96 @@
+let firstNumber;
+let display = document.querySelector('.display')
+display.textContent="";
+
+let clear = document.querySelector('#clear')
+let operators = document.querySelectorAll('.operator')
+let operator;
+let results;
+let secondNumber;
+let getElement = document.querySelectorAll('#click')
```

Comment:
getElement is not a good name. I guess you want to handle the clicked numbers here? Then maybe you should introduce another class so you have `btn number` which you then can assign click handlers to.